src/components/Header.js

```diff
@@ -1,19 +1,16 @@
-import React from 'react';
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
-import { faSearch, faShoppingCart } from '@fortawesome/free-solid-svg-icons'
+import { faSearch, faShoppingCart, faBars } from '@fortawesome/free-solid-svg-icons'
 
-function Header() {
+const Header = (props) => {
```

Comment:
Asi you only have one prop here, well in most cases it's better to destructure it so it's easier to know what props are you getting. In example
```suggestion
const Header = ( { setIsProductsList } ) => {
```

and then where you call it its just 
```
<div className="nav-title" onClick={() => setIsProductsList(false)}>Furnituria</div>
```